src/store/modules/keycodes/index.js

```diff
@@ -1,11 +1,11 @@
-import quantum from './quantum';
 import ansi from './ansi';
 import iso_jis from './iso-jis';
+import quantum from './quantum';
 import settings from './kb-settings';
 import media from './app-media-mouse';
 import steno from './steno';
 const state = {
-  keycodes: [...quantum, ...ansi, ...iso_jis, ...settings, ...media]
+  keycodes: [...ansi, ...iso_jis, ...quantum, ...settings, ...media]
```

Comment:
I feel like we're violating Don't repeat yourself, should probably just assign this once else where as a const and re-use it.

Reply:
This is just restoring what's already on qmk:master. I changed it for development but accidentally committed the change.